sample-data/JSON/users.messages.email.Send.json

```diff
@@ -1 +1 @@
-{"id":"2c404ec3-c8ab-4ff5-bec2-124aef928fd3","user_id":"0123456789abcdef01234567","dispatch_id":{"string":"12345qwert"},"external_user_id":{"string":"855bdb9b-0c50-41c6-bfb7-a9545199445d"},"time":1675114259,"timezone":{"string":"America/Chicago"},"campaign_id":{"string":"09d68677-e8a8-4b34-9851-6031758388aa-filler-events"},"campaign_name":{"string":"Test Campaign 11234567-89ab-cdef-0123-456789abcdef"},"message_variation_id":{"string":"c1234567-89ab-cdef-0123-456789abcdef"},"message_variation_name":{"string":"Test Message Variation"},"email_address":"test@test.com","canvas_id":null,"canvas_name":null,"canvas_variation_id":null,"canvas_variation_name":null,"canvas_step_id":null,"canvas_step_name":null,"send_id":{"string":"f123456789abcdef01234567"},"ip_pool":{"string":"braze_transactional"}}
+{"id":"ac00d6fa-e170-405b-a99a-689826c95bc6","user_id":"0123456789abcdef01234567","dispatch_id":{"string":"12345qwert"},"external_user_id":{"string":"c1bce518-1222-4521-949c-68f106bad7de"},"time":1679949018,"timezone":{"string":"America/Chicago"},"campaign_id":{"string":"c144a91f-e9cb-4da3-a116-e6151caacebf-filler-events"},"campaign_name":{"string":"Test Campaign 11234567-89ab-cdef-0123-456789abcdef"},"message_variation_id":{"string":"c1234567-89ab-cdef-0123-456789abcdef"},"message_variation_name":{"string":"Test Message Variation"},"email_address":"test@test.com","canvas_id":null,"canvas_name":null,"canvas_variation_id":null,"canvas_variation_name":null,"canvas_step_id":null,"canvas_step_name":null,"send_id":{"string":"f123456789abcdef01234567"},"ip_pool":{"string":"braze_transactional"},"message_extras":null}
```

Comment:
Is `null` a good sample value for `message_extras`?

Reply:
No it's not. I'm trying to figure out why that populated incorrectly.